corehq/apps/locations/static/locations/ko/location.js

```diff
@@ -0,0 +1,33 @@
+// for product and user per location selection
+$(function () {
+    $("#id_users-selected_ids").width(800).height(400).multiselect();
+    $("#id_products-selected_ids").width(800).height(400).multiselect();
```

Comment:
curious why width and height are being set in javascript, and if it's important that this specifically be 800x400 regardless of screen size